### PR TITLE
docs(loader): Add S3 and minio to bulk/live loader

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-51

### CODE REVIEW

File: content/deploy/fast-data-loading/bulk-loader.md [hunk 1]
```diff
@@ -143,43 +145,99 @@ $ dgraph bulk -f <file1.rdf, file2.rdf> ...
...
+### Load from S3
+
+To bulk load from Amazon S3, you must have either [IAM](#iam-setup) or the following AWS credentials set
+via environment variables:
+
+ Environment Variable                        | Description
+ --------------------                        | -----------
+ `AWS_ACCESS_KEY_ID` or `AWS_ACCESS_KEY`     | AWS access key with permissions to write to the destination bucket.
+ `AWS_SECRET_ACCESS_KEY` or `AWS_SECRET_KEY` | AWS access key with permissions to write to the destination bucket.
+ `AWS_SESSION_TOKEN`                         | AWS session token (if required).
```
Comment (gja) [inline content/deploy/fast-data-loading/bulk-loader.md:157]: AWS_SESSION_TOKEN, while used, is not really something we should document (unless it's used elsewhere). It's not used except in exceptional scenarios.

File: content/deploy/fast-data-loading/bulk-loader.md [hunk 2]
```diff
@@ -143,43 +145,99 @@ $ dgraph bulk -f <file1.rdf, file2.rdf> ...
...
+
+
+{{% notice "note" %}}
+If the system has access to the S3 bucket, you no longer need to explicitly include these environment variables.  
+{{% /notice %}}
+
+#### IAM setup
```
Comment (gja) [inline content/deploy/fast-data-loading/bulk-loader.md:164]: I don't think there is any point in mentioning this here. It should just be a footnote.

File: content/deploy/fast-data-loading/live-loader.md [hunk 3]
```diff
@@ -36,6 +36,54 @@ $ dgraph live -C -f <path-to-gzipped-RDF-or-JSON-file>
...
+### Load from S3
+
+To live load from Amazon S3, you must have either [IAM](#iam-setup) or the following AWS credentials set
+via environment variables:
+
+ Environment Variable                        | Description
+ --------------------                        | -----------
+ `AWS_ACCESS_KEY_ID` or `AWS_ACCESS_KEY`     | AWS access key with permissions to write to the destination bucket.
+ `AWS_SECRET_ACCESS_KEY` or `AWS_SECRET_KEY` | AWS access key with permissions to write to the destination bucket.
+ `AWS_SESSION_TOKEN`                         | AWS session token (if required).
```
Comment (gja) [inline content/deploy/fast-data-loading/live-loader.md:48]: same here.

File: content/deploy/fast-data-loading/live-loader.md [hunk 4]
```diff
@@ -36,6 +36,54 @@ $ dgraph live -C -f <path-to-gzipped-RDF-or-JSON-file>
...
+
+
+{{% notice "note" %}}
+If the system has access to the S3 bucket, you no longer need to explicitly include these environment variables.  
+{{% /notice %}}
+
+#### IAM setup
```
Comment (gja) [inline content/deploy/fast-data-loading/live-loader.md:55]: ditto here

**Review by gja — APPROVED** [review]

File: content/deploy/fast-data-loading/bulk-loader.md (hunk 1 above)
Comment (bucanero) [reply content/deploy/fast-data-loading/bulk-loader.md]: fixed 👍

File: content/deploy/fast-data-loading/bulk-loader.md (hunk 2 above)
Comment (bucanero) [reply content/deploy/fast-data-loading/bulk-loader.md:158]: fixed

**Pull request merged**